source/install/use-kubernetes-mattermost.rst

```diff
@@ -0,0 +1,89 @@
+.. _use-kubernetes-mattermost:
+
+Using Mattermost Operator Functionality
+======================================
+
+Rolling Upgrades
+----------------
+
+The Mattermost Kubernetes Operator supports rolling upgrades so you can upgrade
+your Mattermost deployment with zero downtime. This process
+requires at least two replicas as a rolling upgrade cannot be performed if there is only one pod.
+Replicas are created when a user count is selected and exceeds 100.
+
+New Mattermost releases are announced via our community server, as well as social media.
+
+**To perform a rolling upgrade:**
+
+1. Log in to your Kubernetes instance.
+2. Open the ``clusterinstallation.yaml`` manifest (the one created during installation).
+3. Update the ``spec.version`` value to the new version.
+4. Save the changes.
+
+Apply the changes with kubectl using
+
+.. code-block:: sh
+
+  $ kubectl apply -n mattermost -f /path/to/cluster-installation.yaml
+
+The operator initiates a job to perform the database migration (if applicable).
```

Comment:
It will always create the job but the job will only perform a DB migration if one is needed